### DISCUSSION TITLE
Desktop Shortcuts

### DISCUSSION THREAD

**Discussion by sigmaphi007** [YerongAI/Office-Tool#631] [Ideas]
I know the app already as create desktop shortcuts. But is it possible to add also to public desktop during installation so all user who log on to pc can see the shortcuts ?

**Comment by YerongAI** [comment]
Already planned as feature request, see #632

**Reply by sigmaphi007** [reply]
Thank you :)

**Reply by YerongAI** [reply]
> Thank you :) 

Already done in version 10.0.6.2.
